### PR TITLE
Feat/artists copyright

### PR DESCRIPTION
**Pull request by pdabrowsky**
https://app.asana.com/0/1202705605335915/1202705607514903/f

### CODE REVIEW

File: src/components/Copyright/Copyright.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+@use '../../styles/breakpoints';
+
+.copyright {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: 1rem;
+  font-family: variables.$font-tertiary;
+  font-weight: variables.$weight-light;
+  margin-bottom: 24px;
```
Comment (dominikx96) [inline src/components/Copyright/Copyright.scss:11]: As I mentioned here: https://github.com/pdabrowsky/react-nowness-training/pull/22#pullrequestreview-1105460237 do not hardcode margin here

File: src/components/Copyright/Copyright.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  @include breakpoints.small-desktop {
+    flex-direction: row;
+    justify-content: space-around;
+    font-size: 1.3rem;
+    margin-bottom: 27px;
+  }
+  @include breakpoints.large-desktop {
+    justify-content: center;
+    gap: 242px;
```
Comment (dominikx96) [inline src/components/Copyright/Copyright.scss:20]: 👀 can you elaborate on why it's this colossal value?

**Review by dominikx96 — CHANGES_REQUESTED** [review]

File: src/components/Copyright/Copyright.scss (hunk 2 above)
Comment (pdabrowsky) [reply src/components/Copyright/Copyright.scss:20]: from figma:
image
it looks like the same with less code then new variant container with hardcoded max-width

**Comment by pdabrowsky** [comment]
Why does the text look different from the design? text and content are the same.

File: src/components/Copyright/Copyright.scss (hunk 2 above)
Comment (dominikx96) [reply src/components/Copyright/Copyright.scss:18]: nope, it's not a good idea to use a gap in this way. You can use the existing container and use `flex` to set proper positions. Figma is not a good source of CSS values :P

**Review by dominikx96 — CHANGES_REQUESTED** [review]
see a comment + resolve conflicts

**Review by dominikx96 — APPROVED** [review]

**Pull request merged**